aristanetworks/atd-public
# Webssh

To Test:
cd /tmp
rm -rf atd
git clone https://github.com/aristanetworks/atd-public.git atd
cd atd
git checkout webssh
sh labvm/services/atdFiles/atdFiles.sh 

To verify:
netstat -an | grep 2222
If it's running, test via http://<public-ip>/ssh/host/192.168.0.14
